packages/api/internal/orchestrator/orchestrator.go

```diff
@@ -80,19 +77,27 @@ func New(
 	return &o, nil
 }
 
-func (o *Orchestrator) Close() error {
-	var err error
-	for _, node := range o.nodes.Items() {
-		closeErr := node.Client.Close()
-		if closeErr != nil {
-			err = errors.Join(err, closeErr)
+func (o *Orchestrator) Close(ctx context.Context) error {
+	nodes := o.nodes.Items()
+	errs := make([]error, 0, len(nodes)+2)
```

Comment:
do you think this is better? I would expect that in 99% it will be empty. Also if somebody adds some new close with potential error, I am little doubtful they will increase the number